enterprise/frontend/src/embedding-sdk-bundle/test/auth-flow/jwt.unit.spec.tsx

```diff
@@ -126,6 +127,26 @@ describe("Auth Flow - JWT", () => {
     );
   });
 
+  it('should not render usage problem popover saying JWT is missing "exp" claim when then token is being fetched', async () => {
+    const customFetchFunction = jest.fn().mockImplementation(() => ({
+      jwt: MOCK_VALID_JWT_RESPONSE,
+    }));
+
+    const authConfig = defineMetabaseAuthConfig({
+      metabaseInstanceUrl: MOCK_INSTANCE_URL,
+      preferredAuthMethod: "jwt",
+      fetchRequestToken: customFetchFunction,
+    });
+
+    setup({ authConfig });
+
+    await waitForLoaderToBeRemoved();
```

Comment:
@npretto I added this line in addition to the test that was added from master.

I'm not sure why, but without this line, it would let unmocked endpoint overflow into the next test 🤷 

Note that this endpoint was actually mocked in this function via `setup`'s chained calls already. It behaves as if it's asserting this call from this test in the next test that's why it throws.

<img width="1333" height="500" alt="Image" src="https://github.com/user-attachments/assets/9b20ef6f-51e6-4972-93fe-a9acac1aec61" />